addons/auth/.template-lintrc.js

```diff
@@ -6,5 +6,5 @@
 'use strict';
 
 module.exports = {
-  extends: ['recommended', 'stylistic'],
+  extends: 'recommended',
```

Comment:
None blocking comment: are we deleting `stylistic` in purpose? in my case, when working on core, it deleted the `'ember-template-lint-plugin-prettier:recommended'` after the update comand, but I restore it since its something we still use/need, wondering if this could be same case.

Reply:
ah yes it was one of the changes i kept from running `ember init` initially i was thinking its not needed but after [looking at the readme](https://github.com/ember-template-lint/ember-template-lint?tab=readme-ov-file#presets) it is needed since this preset is added when `ember-template-lint-plugin-prettier` isn't being used yet. good catch!!